app/scenes/Settings/Tokens.js

```diff
@@ -19,11 +19,11 @@ import Subheading from 'components/Subheading';
 class Tokens extends Component {
   @observable name: string = '';
   props: {
-    settings: SettingsStore,
+    apiKeys: ApiKeySettingsStore,
   };
 
   componentDidMount() {
-    this.props.settings.fetchApiKeys();
+    this.props.apiKeys.fetchApiKeys();
```

Comment:
We should rename this methods to be inline with the others now. `fetchPage` or `fetchAll`